documentation/modules/new-features-and-enhancements-2-10.adoc

```diff
@@ -49,8 +49,13 @@ link:https://issues.redhat.com/browse/MTV-2768[(MTV-2768)]
 
 * *Technology Preview:* In {project-short} 2.10.0, you can live migrate VMs from one OpenShift (OCP) cluster to another, minimizing downtime with the continuous availability of VMs during the migration process. With this update, MTV creates and maps resources from the source to the destination cluster and triggers the live migration, and {virt} CNV migrates the running VM. link:https://issues.redhat.com/browse/MTV-2040[(MTV-2040)]  
 
-* *Technology Preview:* In {project-short} 2.10.0, the Storage Offload Copy feature accelerates disk transfer in the storage array by bypassing the network. This update improves copy speed with the following options:
+* *Developer Preview:* In {project-short} 2.10.0, the Storage Offload Copy feature for warm migration storage offload accelerates disk transfer in the storage array by bypassing the network. This update improves copy speed with the following options for warm migrations:
+
 ** Real-time disk transfer for running VMs. 
 ** Snapshot copies from a running VM.
```

Comment:
I'm still not sure about these two points
we create snapshot and then copy the base disk with the storage offload while the vm is running.
I'm not sure what the "**Real-time** disk transfer" refers to.

** Snapshot copies from a running VM.
we copy snapshots over the network, think this could be ommited.